ui/src/app/shared/job-stream.ts

```diff
@@ -3,12 +3,11 @@ import {BehaviorSubject} from 'rxjs/BehaviorSubject';
 import {JobManagerService} from '../core/job-manager.service';
 import {QueryJobsResponse, QueryJobsResult} from './model/models';
 import {QueryJobsRequest} from "./model/QueryJobsRequest";
+import {initialBackendPageSize} from "./common";
 
 // An observable stream of the client's materialized jobs, where each update
 // contains all jobs that have been loaded so far.
 export class JobStream extends BehaviorSubject<JobListView> {
-  private static readonly minBackendPageSize = 100;
```

Comment:
These are conceptually different, I think it would be better to keep the names separate, even if you just set the value here like `minBackendPageSize = initialBackendPageSize`. One is the initial request size, the other is the minimum total request size (first request or otherwise).